python/cugraph/structure/symmetrize.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cugraph.structure.graph import null_check
+import cudf
+
+
+def symmetrize_df(df, src_name, dst_name):
+    """
+    Take a COO stored in a dataframe, along with the column names of
+    the source and destination columns assumed to represent a directed graph
```

Comment:
Should we get rid of "assumed to represent a directed graph"? It seems to me that a user may see her input as **undirected** (ie. storing only one direction in the edge list) and we would properly return a new graph with directed edges appearing in both directions.
This also appears in `symmetrize` doc.